src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,18 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+    private User user;
```

Comment:
It's better to remove this class field and create local variable for user directly in the method